lib/ruby_lsp/requests/definition.rb

```diff
@@ -26,25 +26,29 @@ class Definition < ExtensibleListener
       sig { override.returns(ResponseType) }
       attr_reader :_response
 
+      # rubocop:disable Metrics/ParameterLists
```

Comment:
After this, we are thinking of introduce some kind of context object to group common parameters together, and reduce the number of things we need to pass around.